.circleci/config.yml

```diff
@@ -103,9 +103,12 @@ workflows:
           - not:
               equal: [ staging, << pipeline.git.branch >> ]
     jobs:
-      - continuation/continue:
-          name: continue-new-branch
-          configuration_path: .circleci/new_branch.yml
+      - path-filtering/filter:
+          name: check-updated-modules-new-branch
+          mapping: |
+            services/database/.* run-validate-database-schema true
+          base-revision: staging  
+          config-path: .circleci/new_branch.yml
```

Comment:
Why are we replacing the continue-new-branch job?

Reply:
We are also continuing from the job here, but can set a flag in case `services/database/` was changed. We only need to run the db validation in that case. CI should still continue in any case on the new_branch workflows.